### DISCUSSION TITLE
News section (LXQt website)

### DISCUSSION THREAD

**Discussion by iamthesenate1** [lxqt/lxqt#2024] [Ideas]
Based on [this](https://github.com/lxqt/lxqt.github.io/issues/67#issuecomment-869048548_) idea by @stefonarch I have made [this](https://github.com/iamthesenate1/lxqt.github.io/blob/News/_posts/2021-06-27-Monthly_News.md)
sketch of a news article summing up all changes since 0.17. 
Opinions (should I continue)?

**Comment by tsujan** [comment]
We announce the changes with every new release. It may seem an easy task but it isn't. Usually, @luis-pereira updates the site based on the changes that are announced for the new releases of individual components. Preparing changelogs and putting the most important changes into release pages on GitHub is my job. Updates are made twice a year, after new LXQt releases are made.

Now, adding a news section for the git changes will require much more manpower, which we really lack. Moreover, why should we announce git changes, while we make release announcements?

However, I may have misunderstood the purpose of a news section. If it isn't about git changes *before the main announcement*, what's it about?

**Reply by iamthesenate1** [reply]
> Now, adding a news section for the git changes will require much more manpower, which we really lack.

I'm willing to write the articles from the news section.

> Moreover, why should we announce git changes, while we make release announcements?

The aforementioned articles would be in their own section so the main page remains uncluttered (maybe just an announcement of the new News section). The sketch is in the main page for now because I wanted to see reactions about this before I make a new section. 

>However, I may have misunderstood the purpose of a news section. If it isn't about git changes before the main announcement, what's it about?

It is for people that don't want to look into every component's commit history to see the changes (people that run git but don't w\ant to recompile every day).

**Reply by stefonarch** [reply]
I was thinking of else than releases, bugfixes and changelogs. which are in "releases" section. But not sure if there are any "real" news. XfCE has a blog section for that, could also be an option, but there has to be content which matters IMO.

**Reply by iamthesenate1** [reply]
> I was thinking of else than releases, bugfixes and changelogs. which are in "releases" section. But not sure if there are any "real" news. XfCE has a blog section for that, could also be an option, but there has to be content which matters IMO.

I understand. It seems that I misunderstand your suggestion.
